gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Forms</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This sections includes all components that are typically used within a form. Components for forms themselves,
+            form structures such as fieldset, and for various kinds of form fields all fall within this section.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Data Presentation</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section groups together components which are intended for organizing and displaying data. This includes
+            views such as lists, tables, and trees, and also visualizations such as the donut chart.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Layout</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Components which make up the "page skeleton" are included in this section. This includes page-level components
+            such as the global sidebar and also containers such as tiles and accordions. In addition, rich examples of
+            layouts incorporating multiple components working together are also found here.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Typography and Icons</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes semantic and typographic containers for text content, such as paragraphs and code
+            blocks.  It also includes utility components that facilitate the use of icons.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>HTML Variants</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Some RSC components and styles have separate support for usage via React component and via bare CSS classes.
+            In these cases, the React components and bare CSS classes are documented separately. For any such component
+            which has both types of documentation, the HTML documentation is currently sequestered in this section.
+            Generally, for React-using consumers, the React components should be preferred over the bare CSS class
+            approach, and this section should thus generally be ignored.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Mixins and Helpers</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes CSS and SCSS utilities such as helper classes, SCSS mixins, and CSS variables.
```

Comment:
This section includes documentation on how to CSS and SCSS utilities like helper classes, SCSS mixins and CSS variables, in a consistent manner.